cg_hermes/config/tags.py

```diff
@@ -91,10 +92,12 @@
     "picard-hs": {"description": "Metrics for the analysis of target-capture sequencing"},
     "picard-insert-size": {"description": "Metrics about the insert size distribution"},
     "picard-wgs": {"description": "Metrics for evaluating the performance of WGS analysis"},
+    "picard-rnaseq": {"description": "Metrics about the alignment of RNA-seq reads"},
     "qc-report": {"description": "Results and QC"},
     "sambamba-depth": {"description": "Coverage information from sambamba"},
     "samtools-stats": {"description": "Comprehensive statistics from alignment file"},
     "software-versions": {"description": "Versions of software used in analysis"},
+    "star": {"description": "Files related to Star aligner"},
```

Comment:
Aren't these two Rnafusion specific?

Reply:
I believe they will be common to rna pipelines - e.g. tomte will also have them